components/SearchResult.tsx

```diff
@@ -1,12 +1,11 @@
-import React from "react";
 import { useQuery } from "@tanstack/react-query";
 import { SearchResponseSchema } from "@/app/utils/discogs.schemas";
 
-export default function SearchResult({ search }: { search: string }) {
+export default function SearchResult({ searchValue }: { searchValue: string }) {
```

Comment:
Best practice is to make the default export the react component the file name references

```suggestion
function SearchResult({ searchValue }: { searchValue: string }) {
```